version.js

```diff
@@ -99,6 +99,14 @@ export async function calculateVersion(context) {
     return localAlphaVersion(nextVersion, timestamp, shortHash);
   }
 
+  if (eventName === "repository_dispatch") {
```

Comment:
Isn't this verbatim the same code than in the `schedule` clause above? Can we merge into one "if schedule or dispatch" branch?

Reply:
I agree on not complicating code with lots of parameters, inheritance and the like for re-use, but if it's literally the same...